.github/workflows/CI.yml

```diff
@@ -67,7 +67,7 @@ jobs:
           # also test `make install` here, as that wants the manual, too;
           # and set NO_COVERAGE=1 to ensure the installed GAP binaries don't
           # contain references to the build directory
-          - os: ubuntu-18.04
+          - os: ubuntu-latest
```

Comment:
This is already in the latest `master`, could you please rebase?

Reply:
done